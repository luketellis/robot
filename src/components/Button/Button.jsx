import React from 'react';

export const Button = ({ label = 'I am a button', ...props }) => {
	return <button {...props}>{label}</button>;
};
