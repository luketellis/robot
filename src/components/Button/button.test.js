import React from 'react';
import { render, cleanup } from '@testing-library/react';
import Button from './Button';

afterEach(cleanup);

it('renders button correctly', () => {
	const { asFragment } = render(<Button />);
	expect(asFragment()).toMatchSnapshot();
});
