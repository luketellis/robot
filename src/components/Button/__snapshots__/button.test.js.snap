// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders button correctly 1`] = `
<DocumentFragment>
  <button
    class="button"
  >
    I am a button
  </button>
</DocumentFragment>
`;
